src/components/Header/Header.css

```diff
@@ -5,25 +5,31 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  color: white;
+  font-size: 10px;
 }
 
 .list-navigation {
   display: flex;
   column-gap: 20px;
   list-style-type: none;
-  color: white;
-  font-size: 10px;
 }
 
 .loupe * {
   fill: white;
 }
 
 .loupe {
-  padding-right: 40px;
   cursor: pointer;
 }
 
 .list-navigation li {
   cursor: pointer;
 }
+
+.right-icon {
```

Comment:
could you choose more meaningful name? does it real right icon?

Reply:
done